src/components/result-pager/result-pager.tsx

```diff
@@ -12,6 +12,17 @@ function ResultPager({
   hitcount
 }: ResultPagerProps) {
   const t = useText();
+
+  if (itemsShown >= hitcount) {
+    return (
+      <div className="result-pager">
+        <p className="text-small-caption result-pager__title">
+          {t("showingText")} {hitcount} {t("outOfText")} {hitcount}
+        </p>
+      </div>
+    );
+  }
+
```

Comment:
I dont understand this part. How can you show more than there is in total? Originally, when I made the first version, I simply hid the pager if there was less items on the page than the pageSize.

Reply:
I've added a comment, I've kept the part that removes the button in the pagination when it is not necessary, and I am setting ```itemsShown``` differently

Reply:
I've fixed this, it still doesnt show the button if there is no reason to click it/no more list items, I dont remember if you had any strong opinions on this matter